src/applications/personalization/profile/components/direct-deposit/EligibilityLink.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import recordEvent from '~/platform/monitoring/record-event';
+
+export const EligibilityLink = ({ typeIsCNP, recordEventImpl }) => {
+  const benefitTypeShort = typeIsCNP ? 'disability' : 'education';
+  const linkText = typeIsCNP
+    ? 'Learn more about disability eligibility'
+    : 'Learn more about GI Bill and other education benefit eligibility';
+
+  return (
+    <a
+      target="_blank"
+      rel="noopener noreferrer"
+      href={`https://www.va.gov/${benefitTypeShort}/eligibility/`}
```

Comment:
just wanted to note that this is a hard-coded link and not relative to env

(disregard if this was intentional!)

Reply:
that is a good point, I had copied that from it's previous usage, but I agree this should be relative to the env. Fix incoming

Reply:
Fixed